app/views/layouts/application.html.erb

```diff
@@ -5,7 +5,7 @@
     <title>BaconCannon =~ An online Ruby parser</title>
     <%= csrf_meta_tags %>
     <%= javascript_pack_tag 'application' %>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <%= stylesheet_pack_tag 'application' %>
```

Comment:
TODO: Is this line needed? 
The file is imported by applicaton.jsx already. https://github.com/pocke/bacon-cannon/pull/37/files#diff-6a65aa8d02d0508c2682762a8145c561R15